### PR TITLE
Update Nuxt to v2.15

### PR DESCRIPTION
**Pull request by abdonrd**
- [x] Use Nuxt v2.15 Typescript built-in support. `nuxt-ts` > `nuxt`
- [x] Remove `@nuxt/typescript-runtime`.
- [x] Remove `ts-node`.
- [x] Update Nuxt Components v2: https://github.com/nuxt/components#migration-guide

---

~~Depends on #2088~~

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -112,7 +112,20 @@ const config: NuxtConfig = {
...
+    '~/components/global',
+    '~/components/landing',
+    '~/components/layouts',
+    '~/components/learn',
+    '~/components/logic',
+    '~/components/metal',
+    '~/components/overview',
+    '~/components/textbook-beta',
+    '~/components/ui'
+  ],
```
Comment (abdonrd) [inline nuxt.config.ts:128]: I would like someone to confirm that this covers all cases.

Comment (y4izus) [reply nuxt.config.ts]: By using this way of register components, nuxt includes prefixes and the nested components are not shown. I've modified the configuration to set the `pathPrefix: false`

**Comment by abdonrd** [comment]
Yay! Seems working now! Thanks @y4izus!

**Review by y4izus — APPROVED** [review]
Thank you @abdonrd !

**Pull request merged**